docs/components/tree-list.md

```diff
@@ -171,7 +171,7 @@ You can pass a `width` string for the entire tree. By default it will take the f
     {{ item.id === 'cats' ? '😸' : item.id === 'bunnies' ? '🐰' : '🐶' }}
   </template>
   <template #item-label="{ item }">
-    <span class="color-purple-400">
+    <span class="slot-color-purple-400">
```

Comment:
Can we rename this to something like `slot-color-purple`? So it doesn't look like utility-class-token thingy